packages/core/src/helpers.ts

```diff
@@ -11,6 +12,7 @@ export interface Config {
   viewsDir: string;
   staticViews: string[];
   webpack?: (defaultConfig: webpack.Configuration, env: 'development' | 'production') => webpack.Configuration;
+  proxyMiddleware?: (config: httpProxy.Config) => httpProxy.Config
```

Comment:
[nits] I want semi colon, too.

If you want to add linting, feel free to send PR :)